tools/gulp/gulpfile.ts

```diff
@@ -1,16 +1,21 @@
+import {createPackageBuildTasks} from './util/package-tasks';
+
+/** Create gulp tasks to build the different packages in the project. */
+createPackageBuildTasks('cdk');
+createPackageBuildTasks('material', ['cdk']);
+createPackageBuildTasks('material-examples', ['material']);
```

Comment:
The name in the `package.json` will be `@angular/material-examples`. Before I went with `material-examples` I had another property on `createPackageBuildTasks` that allows you to specify the task-prefix.

```ts
createPackageBuildTasks('material-examples', 'examples', ['material']);
```
But this just causes confusion because the package and task name are different. Actually the same for `lib` (= `material`).

We should try to be as consistent as possible.